plugins/source/pagerduty/codegen/recipes/schedules.go

```diff
@@ -0,0 +1,13 @@
+package recipes
+
+import "github.com/PagerDuty/go-pagerduty"
+
+func ScheduleResources() []*Resource {
+	return []*Resource{
+		{
+			SubService: "schedules",
+			PKColumns:  []string{"id"},
```

Comment:
all resources have exactly same pk (only `id`).
remove from recipe.Resource (at least, from all recipes), embed in the codegen call

Reply:
I think it's better to be explicit than implicit, and personally am fine with keeping the list of PKs here even if they are all the same

Reply:
I think it's alright to have PK explicitly. especially since it's a small plugin, the one-line-code-duplication optimization is not too important I think.